nixos/tests/unbound.nix

```diff
@@ -133,22 +154,6 @@ import ./make-test-python.nix ({ pkgs, lib, ... }:
           # user that is not permitted to access the unix socket
           unauthorizeduser = {};
         };
-
-        environment.etc = {
```

Comment:
Why did you remove the part that is used to test configuration reloading?

Reply:
Didn't know it was testing that part of the config. Will put it back.